.github/workflows/pull_request.yml

```diff
@@ -10,15 +10,95 @@ on:
       - 19.x
       - 18.x
       - 17.x
+
 jobs:
   buildAndTest:
-    runs-on: ubuntu-latest
+    name: buildAndTest ${{ matrix.java.c }}/${{ matrix.java.t || matrix.java.c }} (${{ matrix.dist.vm }}) on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ 'ubuntu-latest', 'windows-latest' ]
+        java:
+          - { c: '8', t: '8' } # Latest Java 8
+          - { c: '8.0.282', t: '8.0.282' } # Earliest supported Java 8
+          - { c: '11', t: '11' }
+          - { c: '17', t: '17' }
+          - { c: '18', t: '18' }
```

Comment:
Given that 18 is no longer supported, we should be able to drop this and replace it with 17 and 21, and possibly 22

Reply:
now running 11,17,21

I think adding 22-ea in a new PR makes the most sense

Temurin is already releasing EA builds, we just need to make sure Semeru is excluded from the test matrix (or have includes: on 22-ea)